ledger/participant-state/kvutils/app/src/main/scala/com/daml/ledger/participant/state/kvutils/app/Runner.scala

```diff
@@ -43,9 +43,13 @@ final class Runner[T <: ReadWriteService, Extra](
         "[^A-Za-z0-9_\\-]".r.replaceAllIn(name.toLowerCase, "-"))
       implicit val materializer: Materializer = Materializer(actorSystem)
 
+      // FIXME: https://github.com/digital-asset/daml/issues/5164
+      // This should be made configurable
```

Comment:
I'd suggest `Engine()` to return the default configuration -- this way you won't have to change `kvutils` or other code.

Reply:
All the purpose of #5164 is that each implementation of Ledger have to specify *explicitly* which version it allows, so when we introduce a new version in the engine this does not impact the ledger implementations. 
For this reason we definitively do not want a default constructor.